crates/astria-sequencer/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "astria-sequencer"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1"
+async-trait = "0.1.52"
+bytes = "1.4"
+is-terminal = "0.4.7"
+penumbra-component = { git = "https://github.com/penumbra-zone/penumbra.git", tag = "v0.52.2" }
+penumbra-storage = { git = "https://github.com/penumbra-zone/penumbra.git", tag = "v0.52.2" }
+tendermint-proto = "0.29"
+tendermint = "0.29"
+tower = "0.4"
+tower-abci = "0.5.0"
```

Comment:
I had to update tendermint to 0.31 as there was no penumbra release I could find that supported tendermint 0.30. tower-abci was updated to 0.7.0 because of that also

Reply:
we should probably bump the repo-wide tendermint version to 0.31 or 0.32 (the latest)

Reply:
fyi, we cut a release of `tower-abci@0.8` that plugs into `0.32`!

Reply:
nice, will probably do a repo-wide update when this is merged!